src/sysinfo.rs

```diff
@@ -223,7 +224,7 @@ mod test {
             let s = ::System::new();
             assert!(!s.get_name().expect("Failed to get system name").is_empty());
             assert!(!s
-                .get_version()
+                .get_kernel_version()
```

Comment:
Instead of simply replacing, could you check both please?

Reply:
done!